mysql/mysql.go

```diff
@@ -185,8 +187,21 @@ func interfaceToString(face interface{}) (string, error) {
 		}
 	case int:
 		ret = strconv.Itoa(val)
-	case string:
-		ret = val
+	case []uint:
+		length := len(val)
+		if length == 0 {
+			return ret, err
+		}
+		ret = strconv.Itoa(int(val[0]))
```

Comment:
Uint has double the positive range of an int so I would think you'd need to use `strconv.FormatUint(i uint64, base int)` instead. Otherwise you'll be overflowing the int.